src/writeData/subscriptions/components/JsonParsingForm.tsx

```diff
@@ -45,8 +45,10 @@ interface Props {
 
 const JsonParsingForm: FC<Props> = ({formContent, updateForm, edit}) => {
   const stringType = 'String'
-  const numberType = 'Number'
-  const dataTypeList = [stringType, numberType]
+  const floatType = 'Float'
+  const intType = 'Integer'
```

Comment:
I'm trying to think through the value of adding `Integer` 🤔 Since JSON only supports "number", do we really need to specify whether the number in the message has a decimal or not? Or could we get away with using the avro "double" type that we've been using already? 

InfluxDB actually assumes that all numerical values are floats unless you append an `i` to the value: https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_tutorial/#data-types

So I think even if we let the user choose between Float and Integer in the UI, all of those values would still be written as floats to InfluxDB at the end of the day

Reply:
yeah I thinkkkkk the way it works is that the values are all _written_ as floats but a type is inferred when you query it back. Could you repeat the experiment of sending `1` and `1.2` using just `double` as the arvo type and see if notebooks shows the same result?

Reply:
based on [this](https://influxdata.slack.com/archives/C023R2B5KD3/p1658935448594759) conversation i think we should add in the int/float support to be inline with telegraf json v2 parsing